BioArchLinux/r-pmcmr/lilac.yaml

```
@@ -0,0 +1,16 @@
+build_prefix: extra-x86_64
+maintainers:
+- github: kiri2002
+pre_build_script: |
+  for line in edit_file('PKGBUILD'):
+    if line.startswith('_pkgver='):
+      line = f'_pkgver={_G.newver}'
+    print(line)
+  update_pkgver_and_pkgrel(_G.newver.replace(':', '.').replace('-', '.'))
+post_build_script: |
+  git_pkgbuild_commit()
+update_on:
+- source: rpkgs
+  pkgname: pmcmr
```

```suggestion
  pkgname: PMCMR
```
the name is case-sensitive

---

How patient you are!